e2e/tests/api-driven/src/invite-to-pay/helpers.ts

```diff
@@ -3,18 +3,17 @@ import type {
   PaymentRequest,
 } from "@opensystemslab/planx-core/types";
 import axios from "axios";
-import gql from "graphql-tag";
+import { gql } from "graphql-tag";
```

Comment:
This change is repeated a few times - it's due to an issue with how this package is published. Please see https://github.com/apollographql/graphql-tag/issues/804 and https://arethetypeswrong.github.io/?p=graphql-tag%402.12.6